src/OrchardCore.Modules/OrchardCore.Forms/OrchardCore.Forms.csproj

```diff
@@ -4,6 +4,10 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Http.Polly" Version="2.1.0" />
```

Comment:
Move to Dependencies.AspNetCore.props file. See other projects for examples, or ping me.

Reply:
OK

Reply:
That doesn't seem to work - when I do, the Microsoft.Extensions.Http namespace is no longer available. Is there anything else I need to do?